src/components/ProfileTree/ProfileBranch.js

```diff
@@ -113,14 +130,18 @@ export const ProfileBranch = memo(({
                 matchData.map((item, i) => (
                   <ProfileBranch
                     key={`profile-branch-${item.id}-${i}`}
-                    entityKey={`${camelCase(item.contentType)}s`}
-                    recordData={item.content}
-                    contentData={item.childSnapshotWrappers}
-                    record={record}
+                    reactTo="MATCH"
```

Comment:
Consider moving this value to a constant.

Reply:
Done